module/handlebar-helpers.mjs

```diff
@@ -97,4 +96,46 @@ export default function registerHandlebarHelpers() {
   Handlebars.registerHelper( "ed-includes", ( collection, element ) => {
     return Array.from( collection ).includes( element );
   } );
+
+
+  /*************************************************** */
+  /*            Handlebars for Item sheets             */
+  /*************************************************** */
+  Handlebars.registerHelper("isRangedWeapon", function(weaponType) {
+    // Define the logic to determine if the weapon is ranged
+    const rangedWeaponTypes = ["bow", "crossbow", "blowgun", "thrown"]; // Example types
+    return rangedWeaponTypes.includes(weaponType);
+  })
+
+  /*************************************************** */
+  /*            Handlebars for Item Cards              */
```

Comment:
These functions are all used only once in a template and way to specific, "single-use". So they probably shouldn't be helpers but just be added in the corresponding `prepareData` methods (this should generally be the first choice, unless it is something very general and often used)

Reply:
removed handlebars handler and included them in the weapons dataModel as getter and the base-sheet for getData